static/mergify-count-active-users.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+# This scripts counts the number of active users for repositories over the last 30 days.
+# See https://docs.mergify.com/billing/ for context.
+#
+# Keep in mind that this is not 100% accurate, but gives a ballpark estimate.
+#
+# You'll need Python 3 and requests for this to work.
+# You can install requests by running:
+# $ pip install requests
+# then run this script with
+# $ python3 mergify-count-active-users <url of your repo>
+
+import collections
+import requests
+import pprint
+import os
+from datetime import datetime, timedelta
+from urllib.parse import urlparse
+
+LOOKBACK_WINDOW = timedelta(days=30)
+
+HEADERS = {
+    "Authorization": f"token {os.environ['GITHUB_TOKEN']}",
+    "Accept": "application/vnd.github.v3+json",
+}
+
+# Assuming "active" means created in the last N days
+SINCE_DATE = datetime.now() - LOOKBACK_WINDOW
+
+
+def get_api_and_repo_name_from_url(url):
+    parsed = urlparse(url)
+    if parsed.netloc.lower() == "github.com":
+        host = "https://api.github.com"
+    else:
+        host = f"{parsed.netloc}/api/v3"
+
+    return host, parsed.path.strip("/")
+
+
+def get_active_prs_for_repo(api_endpoint, repo_name):
+    page = 1
+    prs = []
+    while True:
+        try:
+            response = requests.get(
+                f"{api_endpoint}/repos/{repo_name}/pulls?per_page=100&page={page}",
+                headers=HEADERS,
+                params={"state": "all"},
+            )
+            response.raise_for_status()
+        except Exception:
+            print("W: Unable to retrieve PR list, ignoring")
+        else:
+            for pr in response.json():
+                created_at = datetime.fromisoformat(pr["created_at"].rstrip("Z"))
+                if created_at >= SINCE_DATE:
+                    prs.append(pr)
+                else:
+                    return prs
+            page += 1
```

Comment:
At first look this will create an infinite loop if there's no PR or if there's no PR created after the limit